modules/nf-core/samtools/sort/main.nf

```diff
@@ -8,11 +8,12 @@ process SAMTOOLS_SORT {
         'biocontainers/samtools:1.19.2--h50ea8bc_0' }"
 
     input:
-    tuple val(meta), path(bam)
+    tuple val(meta) , path(bam)
+    tuple val(meta2), path(fasta)
 
     output:
-    tuple val(meta), path("*.bam"), emit: bam
-    tuple val(meta), path("*.csi"), emit: csi, optional: true
+    tuple val(meta), path("*.{bam,cram}"), emit: sorted
```

Comment:
I think this naming does not follow the guidelines:
<img width="868" alt="delete_me" src="https://github.com/nf-core/modules/assets/6224346/2bf9ae8e-b5e3-4e0d-a3b9-2e036ba4b931">
From [here](https://nf-co.re/docs/contributing/modules#inputoutput-options) point 3.

Reply:
good catch, I'll make the required changes

Reply:
fixed in https://github.com/nf-core/modules/pull/5022/commits/bab0e82566f7d59d1aacdda1d51eba0e28427c1c